SparkPost/2web2ui
# FE-1171 - CircleCI needs new configuration option that controls targeted Cypress tests

[FE-1171](https://sparkpost.atlassian.net/browse/FE-1171)

### What Changed
- Adds new targeted integration test jobs to the CircleCI config

### How To Test
1. Check out this branch, and at the bottom of the CircleCI config, uncomment the targeted tests and introduce some targeted specs like so:
```yml
- 'Integration Testing - OG Theme - Targeted':
    specs: "./cypress/tests/integration/domains/createPage.spec.js"
    requires:
      - 'Build - OG Theme'
- 'Integration Testing - Hibana Theme - Targeted':
    specs: "./cypress/tests/integration/alerts/createPage.spec.js"
    requires:
      - 'Build - Hibana'
```
1. Push up the change and verify the targeted specs are run (you can comment out other steps in the workflow to speed this up if you want)
1. If this works as expected, revert your change to get the branch back to its original state with the targeted workflow steps commented out

### To Do
